### ISSUE TITLE
NSS arrhodes Tesla major flaw

### ISSUE THREAD

**Issue by christianw12313** [ParadiseSS13/Paradise#21442]
**Issue  Description**:
NSS Arrhodes Has a Bug with the Tesla Causing it to arc out of the containment Due to some grounding rods and telsa coils missing.

**What did you expect to happen**: 
the Tesla to stay inside of the containment and produce power without arcing out

**What happened instead**:
the tesla arced out the front of the containment arcing into the Smes causing the Containment field to fail.

**Why is this bad/What are the consequences**:
the Telsa gets loose and Wrecks the station

**Steps to reproduce the problem**:
Start the Tesla on NSS Arrhodes and stand near the window.

**When did the problem start happening**:
1-2 Days ago and the bug still exists

**Extra information**:
adding additional and wellplaced Grounding rods and tesla coils to prevent arcing

**Comment by DGamerL** [comment]
Did it loose before or after the SMES failed?

**Comment by christianw12313** [comment]
> Did it loose before or after the SMES failed?

well due to the flaw in the station the tesla can arc out of its containment and the arc can hit the Smes Causing it to become loose

**Comment by DGamerL** [comment]
Sorry, what I meant to ask was if the tesla could move outside of the containment before the SMES were destroyed

**Comment by christianw12313** [comment]
no but it can shoot arcs out before the containment is gone.

**Comment by christianw12313** [comment]
and those arcs can hit the smes knocking it out

**Comment by DGamerL** [comment]
Ah yeah 100% a mapping issue then, not my TM. Mapper has been notified.

**Comment by kugamo** [comment]
I had moved things around in containment to make singulo more viable, didnt test the new setup with tesla engine though. For the record though on the first round tesla loosed with the new containment, the arcing did happen outside containment.. but the SMES didnt break and containment field was still up @DGamerL. In my testing I did today, the most I could get was arcing in the engine room. This would eventually break the SMES if left alone, or a run of bad arcs... but the tesla warping through a still up containment is not shepards flaw.
